src/main/java/org/gridsuite/study/server/dto/impacts/AbstractBaseImpact.java

```diff
@@ -44,4 +45,10 @@ public enum ImpactType {
     private ImpactType type;
 
     private IdentifiableType elementType;
+
+    @JsonIgnore
+    public abstract boolean isSimple();
+
+    @JsonIgnore
+    public abstract boolean isCollection();
```

Comment:
Why adding these two methods?

Reply:
Asked by @SlimaneAmar  for simplification